.github/workflows/docker-build-test.yaml

```diff
@@ -157,7 +157,30 @@ jobs:
       # Use the cache ID as the Forge namespace so we can limit Forge test concurrency on k8s, since Forge
       # test lifecycle is separate from that of GHA. This protects us from the case where many Forge tests are triggered
       # by this GHA. If there is a Forge namespace collision, Forge will pre-empt the existing test running in the namespace.
-      FORGE_NAMESPACE: forge-${{ needs.determine-docker-build-metadata.outputs.targetCacheId }}
+      FORGE_NAMESPACE: forge-e2e-${{ needs.determine-docker-build-metadata.outputs.targetCacheId }}
+
+  # Run e2e compat test against testnet branch
+  forge-compat-test:
```

Comment:
@rustielin - Can you confirm this test will run in parallel with the existing forge performance test?